### ISSUE TITLE
unexpected zoo in log

### ISSUE THREAD

**Issue by Avorthoren** [sympy/sympy#18034]
I expect to see the results of calculating the fifth and sixth lines of code similar, but I get complex infinity in the first case.
Is it supposed to be like that?
sympy version 1.4

```python
>>> from sympy import log
>>> from sympy.parsing.sympy_parser import parse_expr
>>> finalDist = parse_expr("sqrt(288305443838996749733232910450179213236458916887461997159954)/30563233229717339803725392868")
>>> startDist = parse_expr("4274815913*sqrt(135258220765722511688068035000043802914)/4087643966492512261977499640")
>>> log(finalDist / startDist, 10)
zoo
>>> log(float(finalDist) / float(startDist), 10)
0.367724992542932/log(10)
```

**Comment by Avorthoren** [comment]
Further analysis showed that the problem is integer division
![Screenshot from 2019-12-10 13-19-08](https://user-images.githubusercontent.com/58729518/70525385-e82f4600-1b4f-11ea-8731-dbe2397d2fbd.png)

**Comment by jksuom** [comment]
This has been fixed: https://github.com/sympy/sympy/commit/5b0654201db8930a2c328968ac76acd2dc191e0f#diff-2e916538a3551e5fa46d9f5a46d8d988

**Comment by Avorthoren** [comment]
> This has been fixed: [5b06542#diff-2e916538a3551e5fa46d9f5a46d8d988](https://github.com/sympy/sympy/commit/5b0654201db8930a2c328968ac76acd2dc191e0f#diff-2e916538a3551e5fa46d9f5a46d8d988)

Thank you for your response.
Works fine with sympy 1.5rc1.
When to expect a stable 1.5 release?

**Comment by oscarbenjamin** [comment]
SymPy 1.5 has been released today
